core/storage/skill/gae_models_test.py

```diff
@@ -92,28 +92,28 @@ def setUp(self):
             creator_id='janet_id',
             skill_is_private=True
         ).commit(
-            'janet_commit_id', 'Created new skill rights',
+            'janet_committer_id', 'Created new skill rights',
```

Comment:
> It is different just to test if we can detect both commmitter_id and creator_id separately.

This needs to be explained (in a comment in the code) everywhere you do it. It's not clear to a developer who's casually reading the code.

In addition, consider using something like random_user_id to avoid any implicit association between the two IDs (given the explanation for why you're doing this) -- IIUC, you're trying to test the case where the committer ID and the creator ID don't necessarily represent the same person for some reason, right? I just don't want to break the conceptual baseline that "one user == one user_id".

Reply:
I have changed the naming of the user ids altogether and added a comment.